Ludo/Game/GameController.cs

```diff
@@ -179,7 +179,7 @@ public ZoneType GetZoneType(int x, int y)
         public bool IsSafeZone(int x, int y)
         {
             var zone = GetZoneType(x, y);
-            return zone == ZoneType.SafeZone || zone == ZoneType.StartPoint || zone == ZoneType.HomePath;
+            return zone == ZoneType.SafeZone || zone == ZoneType.StartPoint || zone == ZoneType.HomePath; 
```

Comment:
Return value instead of di-hardcode, better dimasukin variable baru variablenya di-return

```
bool isSafe = false;

isSafe = zone == ZoneType.SafeZone || zone == ZoneType.StartPoint || zone == ZoneType.HomePath;

return isSafe;
```

Atau biar ga kepanjangan
```
bool isSafe = zone == ZoneType.SafeZone;
isSafe |=  zone == ZoneType.StartPoint;
isSafe |= zone == ZoneType.HomePath;

return isSafe;
```